xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -111,10 +129,19 @@
         <Parameter Name="source" Type="System.ReadOnlySpan&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">The read-only span to read.</param>
+        <summary>Reads an <see cref="T:System.Int32" /> out of a read-only span of bytes as big endian.</summary>
```

Comment:
Find/Replace `bytes as big endian` with `bytes, as big endian` and `bytes as little endian` with `bytes, as little endian`
```suggestion
        <summary>Reads an <see cref="T:System.Int32" /> out of a read-only span of bytes, as big endian.</summary>
```